src/clue/adafruit_clue.py

```diff
@@ -251,10 +253,168 @@ def were_pressed(self):
           while True:
               print(clue.were_pressed)
         """
-        ret = self.__pressed_buttons.copy()
-        self.__pressed_buttons.clear()
+        ret = self.__state["pressed_buttons"].copy()
+        self.__state["pressed_buttons"].clear()
         return ret
 
+    @property
+    def acceleration(self):
+        """Obtain acceleration data from the x, y and z axes.
+        This example prints the values. Try moving the board to see how the printed values change.
+        To use with the CLUE:
+        .. code-block:: python
+          from adafruit_clue import clue
+          while True:
+            print("Accel: {:.2f} {:.2f} {:.2f}".format(*clue.acceleration))
+        """
+        return (
+            self.__state["acceleration"]["x"],
+            self.__state["acceleration"]["y"],
+            self.__state["acceleration"]["z"],
+        )
+
+    @property
+    def color(self):
+        """The red, green, blue, and clear light values. (r, g, b, c)
+        This example prints the values. Try holding something up to the sensor to see the values
+        change. Works best with white LEDs enabled.
+        To use with the CLUE:
+        .. code-block:: python
+          from adafruit_clue import clue
+          while True:
+              print("Color: R: {} G: {} B: {} C: {}".format(*clue.color))
+        """
+        return (
+            self.__state["color_sensor"]["r"],

```

Comment:
I was wondering if string keys like `'color_sensor'` could be extracted into a class containing only constants. Seems like if we were to change the key name in the future, we would have to change the string at a bunch of places.

Reply:
honestly very good point. I'll update that 👍